src/main/java/id/co/veritrans/mdk/v1/VtGatewayConfig.java

```diff
@@ -14,6 +14,18 @@
     private final String clientKey;
     private final int maxConnectionPoolSize;
     private final ProxyConfig proxyConfig;
+    private int connectTimeout = DEFAULT_CONNECT_TIMEOUT;
```

Comment:
maybe we can move this part to `src/main/java/id/co/veritrans/mdk/v1/VtGatewayConfigBuilder.java` since all the default value is setup on the builder.

so we can remove `connectTimeout`, `socketTimeout`, `DEFAULT_CONNECT_TIMEOUT`, and `DEFAULT_SOCKET_TIMEOUT` from this file.